x-pack/plugins/enterprise_search/public/applications/workplace_search/constants.ts

```diff
@@ -54,4 +54,194 @@ export const SOURCE_STATUSES = {
   ALWAYS_SYNCED: 'always_synced',
 };
 
+export const SOURCE_NAMES = {
+  CONFLUENCE: i18n.translate(
+    'xpack.enterpriseSearch.workplaceSearch.sources.sourceNames.confluence',
+    { defaultMessage: 'Confluence' }
+  ),
+  CONFLUENCE_SERVER: i18n.translate(
+    'xpack.enterpriseSearch.workplaceSearch.sources.sourceNames.confluenceServer',
+    { defaultMessage: 'Confluence (Server)' }
+  ),
+  DROPBOX: i18n.translate('xpack.enterpriseSearch.workplaceSearch.sources.sourceNames.dropbox', {
+    defaultMessage: 'Dropbox',
+  }),
+  GITHUB: i18n.translate('xpack.enterpriseSearch.workplaceSearch.sources.sourceNames.github', {
+    defaultMessage: 'GitHub',
+  }),
+  GITHUB_ENTERPRISE: i18n.translate(
+    'xpack.enterpriseSearch.workplaceSearch.sources.sourceNames.githubEnterprise',
+    { defaultMessage: 'GitHub Enterprise Server' }
+  ),
+  GMAIL: i18n.translate('xpack.enterpriseSearch.workplaceSearch.sources.sourceNames.gmail', {
+    defaultMessage: 'Gmail',
+  }),
+  GOOGLE_DRIVE: i18n.translate(
+    'xpack.enterpriseSearch.workplaceSearch.sources.sourceNames.googleDrive',
+    { defaultMessage: 'Google Drive' }
+  ),
+  JIRA: i18n.translate('xpack.enterpriseSearch.workplaceSearch.sources.sourceNames.jira', {
+    defaultMessage: 'Jira',
+  }),
+  JIRA_SERVER: i18n.translate(
+    'xpack.enterpriseSearch.workplaceSearch.sources.sourceNames.jiraServer',
+    { defaultMessage: 'Jira (Server)' }
+  ),
+  ONE_DRIVE: i18n.translate('xpack.enterpriseSearch.workplaceSearch.sources.sourceNames.oneDrive', {
```

Comment:
Just for our own sanity, I suggest that we use the same spacing as an official name: ONE_DRIVE -> ONEDRIVE
Same for SERVICE_NOW and SHARE_POINT.

Reply:
Thanks for updating these! I think there's still SHARE_POINT left.

Reply:
Yep! Sorry about that https://github.com/elastic/kibana/pull/83459/commits/4b102fc8afc9481a132f23cce88f9593ea4b9ebd